### PR TITLE
Add CI job to build Python wheels

### LINKED ISSUE #2: Wheels on PyPI
Hi. Thanks for the great project!

Have you considered publishing wheels on PyPI so users don't have to compile the extension from sources? It might be useful mainly for containerized applications where you don't have packages like sqlite-devel libicu-devel or gcc installed.

If you want to, I can help you with that.

### PR DESCRIPTION
**Pull request by frenzymadness**
The first draft of the GH action. You can see its results here: https://github.com/frenzymadness/sqlite-icu/actions/runs/2483703324

FIxes: #2

### CODE REVIEW

File: .github/workflows/build_wheels.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      - uses: actions/setup-python@v2
+
+      - name: Install cibuildwheel
+        run: python -m pip install cibuildwheel==2.6.1
+
+      - name: Build wheels
+        run: python -m cibuildwheel --output-dir wheelhouse
+        env:
+          CIBW_BEFORE_ALL_LINUX: yum install -y libicu-devel sqlite-devel
+          CIBW_SKIP: "*-musllinux_*"
```
Comment (karlb) [inline .github/workflows/build_wheels.yml:22]: Any specific reason for skipping musl?

**Review by karlb — APPROVED** [review]
Looks good, thanks! I'll see if I can upload the wheels to the existing release.

File: .github/workflows/build_wheels.yml (hunk 1 above)
Comment (frenzymadness) [reply .github/workflows/build_wheels.yml:22]: I don't have any experience with musllinux and I'd have to find a way how to install the devel packages there and their correct names. I'd expend this per request.

Comment (karlb) [reply .github/workflows/build_wheels.yml:22]: It's fine as it is. I was just curious.

**Pull request merged**